tests/models/test_lora_layers.py

```diff
@@ -464,6 +464,14 @@ def test_lora_unet_attn_processors_with_xformers(self):
                 if isinstance(module, Attention):
                     self.assertIsInstance(module.processor, LoRAXFormersAttnProcessor)
 
+            # unload lora weights
+            sd_pipe.unload_lora_weights()
```

Comment:
I wasn't able to run this test as is (although I've confirmed the fix works with testing it on a production application of ours). If anyone has a machine with the proper requirements and can run it to ensure it is not flaky, i'd be super glad (and if it is failing, let me know, i'll try my best to fix it)!

Reply:
Tested with `RUN_SLOW=1 pytest python tests/models/test_lora_layers.py` and it passed. So, I guess we're good here.